Source/WebCore/page/LocalFrameView.cpp

```diff
@@ -3052,6 +3052,13 @@ bool LocalFrameView::isRubberBandInProgress() const
     return false;
 }
 
+bool LocalFrameView::isInStableState() const
+{
+    if (auto* page = m_frame->page())
+        return page->chrome().client().isInStableState();
+    return ScrollableArea::isInStableState();
```

Comment:
Is it reasonable to jump all the way to `ScrollableArea` here? Should we just call into the direct base class in case we need other stability logic in intervening layers?

Reply:
Whoops, good catch! This is just a typo — I intended to just call into the immediate base class.